src/meta/src/hummock/compaction/picker/trivial_move_compaction_picker.rs

```diff
@@ -24,18 +24,21 @@ pub struct TrivialMovePicker {
     level: usize,
     target_level: usize,
     overlap_strategy: Arc<dyn OverlapStrategy>,
+    enable: bool,
 }
 
 impl TrivialMovePicker {
     pub fn new(
         level: usize,
         target_level: usize,
         overlap_strategy: Arc<dyn OverlapStrategy>,
+        enable: bool,
```

Comment:
Can we just not initialize the picker when `enable==false` outside instead of passing a enable flag to the picker and letting each method return None?

Reply:
Fix in the latest commit.